test/test_descriptor.js

```diff
@@ -34,3 +37,118 @@ describe('generate dp', () => {
     })
   })
 })
+
+describe('generate datapackage.json from local files: single file', () => {
+  it('check for csv data schema', function() { 
```

Comment:
ES6 style thing here as well. Thin we can use `async/await` instead of `.then()`
```
 it('check for csv data schema', async () => {  // here on line 42 and
  const out = await generateDescriptorForLocalFiles()  // down on line 83
  assert.deepEqual(out, expectedSchema)   // and 84
  ..
}
```